posts/2016-12-15-serverless-v1.4.0.md

```diff
@@ -0,0 +1,146 @@
+```
+
+#### Install a plugin
+
+```bash
+serverless plugin install --name serverless-webpack
+```
+
+#### Uninstall a plugin
+
+```bash
+serverless plugin uninstall --name serverless-webpack
+```
+
+This makes it a lot easier to find and manage Serverless plugins for your service
+
+**Note:** You're a plugin author or found a Serverless plugin which is misisng? Feel free to submit a PR to the [Serverless plugin registry](https://github.com/serverless/plugins/edit/master/plugins.json)!
+
+### Alexa event support
+
+v1.4 also introduces a new event `alexaSkill` event source.
+
+You can now hook up your Lambdas with `alexaSkill` events like this:
+
+```yml
+functions:
+  hello:
+    handler: handler.hello
+    events:
+      - alexaSkill: true
+```
+
+You can read more in [the docs](https://serverless.com/framework/docs/providers/aws/events/alexa-skill) about this new event source.
+
+### C# service template
+
+AWS introduced Dotnet core support for Lambdas at re:Invent.
+
+Serverless has you covered and ships with the new `aws-csharp` template you can use to create C# services on AWS Lambda.
+
+Simply run
+
+```bash
+serverless create --template aws-csharp
+```
+
+to create a new C# service and you're good to go!
+
+### Directory support for service installation via git
+
+The [Serverless examples repository](https://github.com/serverless/examples) is the go to point if you want to see some real world examples and learn how Serverless architectures can look like.
+
+The `serverless install` command makes it possible for you to download services based on git repository URLs. The downside was that only full repositories could be downloaded.
+
+Until now! A recent enhancement makes it possible to download specific directories from a git URL.
+
+Want to play around with the REST API example which connects to a DynamoDB?
+
+Just install the service with the following command:
+
+```bash
+serverless install --url https://github.com/serverless/examples/tree/master/aws-node-rest-api-with-dynamodb
+```
+
+### Service wide metrics
+
+v1.3 introduced metrics support which means that you could get function metrics by running:
+
+```bash
+serverless metrics --function hello
+```
+
+v1.4 enhances this command so that you can get service wide metrics (all functions combined) when you run:
+
+```
+serverless metrics
+```
+
+### Stdin support for invoke
+
+`serverless invoke` and it's local equivalent `serverless invoke local` are a great way to invoke your function to test it's behavior.
+
+v1.4 now supports stdin so that you can pipe data directly into the `serverless invoke` call like this:
+
+```bash
+echo "hello world!" | serverless invoke --function hello --stage dev --region us-east-1
+```
+
+This gives you way more flexibility to invoke a function with different data. Here's an example how a `dataGenerator.js` file could be used to generate data for your function:
+
+```bash
+dataGenerator.js | serverless invoke local --function hello
```

Comment:
Good find. This should be updated as well:

https://github.com/serverless/serverless/blob/master/docs/providers/aws/cli-reference/invoke.md#function-invocation-with-data-from-standard-input

https://github.com/serverless/serverless/blob/master/docs/providers/aws/cli-reference/invoke.md#function-invocation-with-data-from-standard-input